src/components/Folder/FolderList.js

```diff
@@ -10,7 +12,7 @@ const FolderItem = ({ data, onFolderClick, isSelected }) => {
   };
   return (
     <div
-      className={`folder-button ${isSelected ? "selected" : ""}`}
+      className={`folder-button ${isSelected ? 'selected' : ''}`}
```

Comment:
이런 형태가 많아진다면,
classNames 라는 라이브러리 사용을 고려해보세요!
https://www.npmjs.com/package/classnames

아래와 같이 코드가 직관적이고 깔끔해집니다.
```suggestion
      className={classNames('folder-button', { selected: isSelected })}
```